### PR TITLE
🇯🇵 Add reiwa

### PR DESCRIPTION
**Pull request by risacan**
Add new Japanese era "reiwa" "令和" which starts from 2019.5.1

### CODE REVIEW

**Comment by tomiacannondale** [comment]
Thank you PR!
Too fast!
I will check PR!

File: lib/era_ja/conversion.rb [hunk 1]
```diff
@@ -39,8 +40,10 @@ def to_era(format = "%o%E.%m.%d", era_names: ERA_NAME_DEFAULTS)
           str_time = era_year(year - 1911, :taisho, era_names)
         when self.to_time < ::Time.mktime(1989,1,8)
           str_time = era_year(year - 1925, :showa, era_names)
-        else
+        when self.to_time < ::Time.mktime(2019, 5, 1)
           str_time = era_year(year - 1988, :heisei, era_names)
+        else
+          str_time = era_year(year - 2019, :reiwa, era_names)
```
Comment (MITSUBOSHI) [inline lib/era_ja/conversion.rb]: @risacan 
It seems to me that `2018` is correct.
If so, please modify it and I hope some spec will be added.

```ruby
irb(main):006:0> Date.new(2019,5,1).to_era
=> "R00.05.01"
```

Comment (risacan) [reply lib/era_ja/conversion.rb]: Thank you ! I will fix this 👍

Comment (tomiacannondale) [reply lib/era_ja/conversion.rb]: Thank you comment! (@MITSUBOSHI )
You are right!

@risacan 
Can you fix it?
and Can you add some tests?

Comment (risacan) [reply lib/era_ja/conversion.rb]: @tomiacannondale 
I fixed the year and add some test. Can you check that? 😄  
Please tell me if you need more tests! 💪

Comment (risacan) [reply lib/era_ja/conversion.rb]: oh i didnt notice the conflict! 😱  I'll fix that

Comment (risacan) [reply lib/era_ja/conversion.rb]: @tomiacannondale 
I fixed some conflict + Add spec for the first day of reiwa and after.

Comment (tomiacannondale) [reply lib/era_ja/conversion.rb]: Great 😄 

But tests for Time were not added 😢 

Can you fix it?

Comment (risacan) [reply lib/era_ja/conversion.rb]: Okay! 👍  I'll add them!

File: spec/time_spec.rb [hunk 2]
```diff
@@ -16,7 +16,7 @@
     end
 
     context 'time is 2019,4,30' do
-      subject { Date.new(2019,4,30) }
+      subject { Time.mktime(2019,4,30) }
```
Comment (risacan) [inline spec/time_spec.rb]: I fixed this because it is a spec for time 🕐

File: spec/time_spec.rb [hunk 3]
```diff
@@ -5,8 +5,18 @@
...
+    end
+
+    context 'time is 2019,5,1' do
+      subject { Time.mktime(2019,5,1) }
+      include_examples "2019,5,1"
+    end
+
     context 'time is 2019,4,30' do
-      subject { Date.new(2019,4,30) }
+      subject { Time.mktime(2019,4,30) }
```
Comment (risacan) [inline spec/time_spec.rb]: I fixed this because it is a spec for time 🕐

Comment (tomiacannondale) [reply spec/time_spec.rb]: Thank you!

I overlooked it 😢

File: lib/era_ja/conversion.rb (hunk 1 above)
Comment (risacan) [reply lib/era_ja/conversion.rb]: @tomiacannondale 
Added time spec for the first day of reiwa and after ✔️ Can you check these? 🙏

**Comment by tomiacannondale** [comment]
Thank you PR and review!

I will update version and release new version!

I looking for you next contributing 😄

**Comment by risacan** [comment]
Thank you so much! 😆

**Pull request merged**